### ISSUE TITLE
Dart_SDK.xml not correctly updated

### ISSUE THREAD

**Issue by gerritwitkamp** [pingbird/puro#29]
Hi,

I have installed some environments, output of puro ls:
```
    * env_281  (stable / 2.8.1 / 77d935af4d)
      env_3105 (stable / 3.10.5 / 796c8ef792)
      env_3106 (stable / 3.10.6 / f468f3366c)
```

When I execute ```puro use env_281 --intellij``` it updates my Dart_SDK.xml (within the .idea/libraries folder) correctly:

```
<component name="libraryTable">
  <library name="Dart SDK">
    <CLASSES>
      <root url="file:///$USER_HOME$/.puro/envs/env_281/flutter/bin/cache/dart-sdk/lib/async"/>
      ...
```

When I execute any other version, like ```puro use env_3105 --intellij``` for example, my Dart_SDK.xml becomes:
```
<component name="libraryTable">
  <library name="Dart SDK">
    <CLASSES/>
    <JAVADOC/>
    <SOURCES/>
  </library>
</component>
```

The Dart SDK path and Flutter SDK path in the settings of my Android Studio are empty also.
I'm using Puro 1.3.4 (distribution/windows-x64)

Any idea what's going wrong here?

**Comment by pingbird** [comment]
I've seen the Intellij plugin freak out and disable itself, perhaps it could be that. Looking into it.

**Comment by gerritwitkamp** [comment]
thanks, like to hear from you

**Comment by pingbird** [comment]
This should be fixed in Puro 1.3.6
